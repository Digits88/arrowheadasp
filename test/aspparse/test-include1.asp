
Here is a test of including files.

<!-- #include file="test-include1.inc" -->

