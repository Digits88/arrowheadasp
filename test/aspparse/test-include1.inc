Here is the include file.

<%
Response.Write("Hello from the include file.")
%>
