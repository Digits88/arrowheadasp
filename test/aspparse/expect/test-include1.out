
Here is a test of including files.

Here is the include file.

Hello from the include file.
